apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabTheorycraft/optimizeTc.ts

```diff
@@ -237,6 +220,23 @@ export function optimizeTcUsingNodes(
       }
       const result = results[0]
```

Comment:
why don't we change the flow a bit more? 

```
compute()
if(result <= max) return
check constraints return
check feasibility return
```

I guess my concern is the case where `compute` is more expensive than the feasibility check.

Reply:
I tested both Hu Tao `8 N2C` and Cyno `4TF Quickbloom` and they were both faster in this PR. On which multitarget do you think `compute` is more expensive?

Reply:
Can you try the same with moving the constraint check after the max check? wonder if that is faster as well

Reply:
To answer your previous question, I think we probably shouldn't cater to crazy mtargets, this solution is good.